backend/routes/recipe.js

```diff
@@ -0,0 +1,76 @@
+const Recipe = require('../models/recipeSchema');
+const express = require('express');
+
+const router = express.Router();
+
+// get all recipes
+router.get('/', async (req, res) => {
+	const recipes = await Recipe.find({});
+	res.send(recipes);
+});
+
+// get specific recipe
+router.get('/:recipeName', async (req, res) => {
+	const recipeName = req.params.recipeName;
+	const recipe = await Recipe.findOne({
+		recipeName: recipeName
+	});
+	res.send(recipe);
+});
+
+// create new recipe (given a body)
+router.post('/', async (req, res) => {
+	const { recipeName, recipeImage, ingredientList, steps } = req.body;
+	let recipe = new Recipe({
+		recipeName,
+		recipeImage,
+		ingredientList,
+		steps
+	});
+
+	await recipe.save((err, recipe) => {
+		// error handling for POST request
+		if (err) {
+			return next(err);
+		}
+		res.send(`Recipe called ${recipeName} added to collection`);
+	});
+});
+
+// adds ingredient to a given recipe
```

Comment:
This is an example of a comment that is very beneficial to have, it explains what your request does